pixi.toml

```diff
@@ -1,20 +1,41 @@
 [project]
-name = "gluex-nstar"
-version = "0.1.0"
-description = "Add a short description here"
 authors = ["Vitor Shen <17490173+shenvitor@users.noreply.github.com>"]
 channels = ["conda-forge"]
+description = "Add a short description here"
```

Comment:
I think it's fine to leave out `version` and `description` for now. They're optional
https://pixi.sh/latest/reference/configuration/#version-optional